src/expression-to-formula.js

```diff
@@ -50,7 +52,7 @@ export default function expressionToFormula(expression) {
     return `${args.join(operator)}`;
   }
 
-  if (symbolOperatorRegExp.test(operator)) {
+  if (/^[^a-zA-Z]/.test(operator)) {
```

Comment:
This regex (nor the previous, but I didn't catch till now) doesn't work quite as expected for the negation operator `!`, which only takes one argument. I suppose the formula-ized version should be `!("foo")`.